### PR TITLE
CIP-0017 | Adjust preamble and structure w.r.t CIP-0001

### LINKED ISSUE #661: CIP-0017 | Adjust preamble and structure w.r.t CIP-0001
Fixed by #662.

### PR DESCRIPTION
**Pull request by rphair**
Fixes #661.

([edited proposal](https://github.com/rphair/CIPs/blob/remediate-0017/CIP-0017/README.md))

### CODE REVIEW

**Comment by rphair** [comment]
@KtorZ when we were originally reviewing this it seemed like this was on the threshold of implementation, especially since Adafolio was already half way there, and maybe that's why it was merged as `Active` without the evidence that we would demand today.

Based on our current standards I've demoted it to `Proposed` but if can think of anything that qualifies it as `Active` then let's have another look.  I was also liberal about writing your _Path to Active_ based on what we might expect along those lines & would be open to your suggestions or corrections.

**Review by Ryun1 — APPROVED** [review]
LGTM

File: CIP-0017/README.md [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
 Title: Cardano Delegation Portfolio
-Authors: Matthias Benkort <matthias.benkort@iohk.io>
-Comments-Summary: A Common format for sharing Cardano delegation portfolios
-Comments-URI: https://github.com/cardano-foundation/CIPs/wiki/Comments:CIP-0017
-Status: Active
-Type: Standards
+Status: Proposed
+Category: Tools
+Authors:
+  - Matthias Benkort <matthias.benkort@iohk.io>
```
Comment (KtorZ) [inline CIP-0017/README.md:7]:
```suggestion
  - Matthias Benkort <matthias.benkort@cardanofoundation.org>
```

File: CIP-0017/README.md [hunk 2]
```diff
@@ -1,24 +1,26 @@
 ---
 CIP: 17
 Title: Cardano Delegation Portfolio
-Authors: Matthias Benkort <matthias.benkort@iohk.io>
-Comments-Summary: A Common format for sharing Cardano delegation portfolios
-Comments-URI: https://github.com/cardano-foundation/CIPs/wiki/Comments:CIP-0017
-Status: Active
-Type: Standards
+Status: Proposed
```
Comment (KtorZ) [inline CIP-0017/README.md:4]:
```suggestion
Status: Inactive (abandoned for lack of interest)
```

**Comment by rphair** [comment]
Still @KtorZ I think it'll be helpful as a point of reference since the idea of the JSON portfolio has come up in other arrangements (e.g  #628) & will keep coming up.  I've changed the status... does anything else need to be reconsidered?

**Review by KtorZ — APPROVED** [review]

**Pull request merged**